File: codes/php/Array.php
<?php

$my_fruits_in_fridge = ["apple", "mango", "orange"];

echo("\narray length: " . count($my_fruits_in_fridge));

foreach ($my_fruits_in_fridge as $array_item_index => $array_item) {
    echo("\n" . $array_item_index);
}

// array length: 3
// 0
// 1
// 2

File: codes/go/Array/Array.go
package main

import "fmt"

const EMPTY_STRING = ""
const TAB = "    "

type Any interface{}
type Object map[string]Any

func main() {
	myFruitsInFridge := []Any{"apple", "mango", "orange"}

	fmt.Println("array length:", len(myFruitsInFridge))

	for arrayItemIndex, _ := range myFruitsInFridge {
		fmt.Println(arrayItemIndex)
	}

	// array length: 3
	// 0
	// 1
	// 2
}
